validator/validators/stress/stress_validator.go

```diff
@@ -113,12 +113,12 @@ var (
 				"net_packets_sent":     float64(700),
 			},
 			"logs": {
-				"procstat_cpu_usage":   float64(225),
-				"procstat_memory_rss":  float64(440000000),
+				"procstat_cpu_usage":   float64(400),
```

Comment:
is this 400% CPU. Out of how many cores?

Reply:
Out of 4 cores based on the[t3a.xlarge](https://instances.vantage.sh/aws/ec2/t3a.xlarge) and yes, its depends on the instance size